release-calendar/.gcloudignore

```diff
@@ -0,0 +1,5 @@
+*
+!app.yaml
+!dist/**
```

Comment:
How would you feel about being more explicit about the ignores, instead of the inverted list? ie

```gitignore
*.ts
*.config.js
.*
{package-lock,ts-config}.json
cloud_build.yaml
```

Reply:
I kind of like it the original way, it tells you exactly what's being deployed.

Reply:
Feels kinda weird to me, but up to you :+1: